docs/dev/search-integration.rst

```diff
@@ -1,7 +1,7 @@
 Server Side Search Integration
 ==============================
 
-Read the Docs provides :doc:`server side search (SSS) <rtd:server-side-search>`
+Read the Docs provides :doc:`server side search (SSS) <server-side-search>`
```

Comment:
This was pointing to the feature on the user docs.

```suggestion
Read the Docs provides :doc:`server side search (SSS) <rtd:server-side-search/index>`
```

Reply:
Ha, OK. We commented the same at exactly the same time 👍🏼 . Perfect!